gtsam/linear/GaussianBayesNet.h

```diff
@@ -228,6 +234,14 @@ namespace gtsam {
      * @return The determinant */
     double logDeterminant() const;
 
+    /**
+     * @brief Get the log of the normalization constant corresponding to the
+     * joint Gaussian density represented by this Bayes net.
+     *
+     * @return double
+     */
+    double logNormalizationConstant() const;
```

Comment:
No unit test

Reply:
Good catch. I'm updating a unit test in `testGaussianBayesNet.cpp` that will test this.